web/app.php

```diff
@@ -3,7 +3,7 @@
 use Symfony\Component\HttpFoundation\Request;
 
 /**
- * @var \Symfony\Component\ClassLoader\ClassLoader
+ * @var \Composer\Autoload\ClassLoader
  */
 $loader = require __DIR__.'/../app/autoload.php';
```

Comment:
`$loader` even looks unused, so the variable could be skipped entirely.

Reply:
`$loader` assignments removed